diagnostic_feedback/templates/student/quiz.html

```diff
@@ -0,0 +1,53 @@
+<div class="diagnostic-feedback diagnostic-feedback-student">
+    <div class="msg"><h3></h3></div>
+    <form id="student_view_form">
+        {% if user_is_staff %} <a href="#" id="export_data">Download Data</a><span id="export_progress"></span> {% endif %}
+        {% if questions %}
+            <h1 class="quiz-title"><i class="icon fa fa-list"></i> {{self.title}} </h1>
+            <h3 class="quiz-description">{{self.description|safe}}</h3>
+            <div class="question-div">
+                <input type="hidden" name="completed_step" id="completed_step" value="{{self.current_step}}">
+                {% for question in questions %}
+                <h3 style="display:none;">Question</h3>
+
+                <section class="answer-choice question-container">
+                    <div class="question-title">{{ question.title }} </div>
+                    <div class="quiz-question">{{ question.text|safe }} </div>
+                    <div class="user-answers">
+                        {% for choice in question.choices %}
+                             {% if self.quiz_type == self.BUZ_FEED_QUIZ_VALUE %}
+                                 <input style="display : inline" class="required" type="radio" name="choice_name_{{ question.id}}"
+                                    {% if question.student_choice == choice.category_id %}
+                                        checked="checked"
+                                    {% endif %}
+                                    value="{{ choice.category_id}}" />
+                             {% else %}
+                                   <input style="display : inline" class="required" type="radio" name="choice_name_{{ question.id}}"
+                                    {% if question.student_choice == choice.value  %}
+                                        checked="checked"
+                                    {% endif %}
+                                    value="{{ choice.value }}" />
+                            {% endif %}
+                            <label>{{choice.name }}</label> <br>
```

Comment:
Clicking on the `label` element should select that choice, but it doesn't. You need to add a `for` attribute or wrap the `<label>` tag around the `<input>` element.